### ISSUE TITLE
Supported Go version in readme

### ISSUE THREAD

**Issue by tmaiaroto** [aws/aws-xray-sdk-go#24]
Just a quick note. I don't think this is compatible with Go 1.7.x

I was using 1.7.4 and saw this error:
```
# github.com/aws/aws-xray-sdk-go/xray
../../aws/aws-xray-sdk-go/xray/httptrace.go:193: unknown httptrace.ClientTrace field 'TLSHandshakeStart' in struct literal
../../aws/aws-xray-sdk-go/xray/httptrace.go:196: unknown httptrace.ClientTrace field 'TLSHandshakeDone' in struct literal
```

Updated to latest stable 1.9.1 and the errors went away.

**Comment by luluzhao** [comment]
@tmaiaroto , thank you for reaching out to us. We checked the golang `httptrace` library and the `TLSHandshakeStart` and `TLSHandshakeDone` functions are only available for Go 1.8+. This is the cause of your problem, since you were using Go 1.7.4. If you want to trace information for `TLSHandshakeStart` and `TLSHandshakeDone`, please update your current Go version.
 
If you are required to use Go 1.7.4,  we will fix the issue in our next release of the Go AWS X-Ray SDK. Thank you for bringing this to our attention.

**Comment by luluzhao** [comment]
@tmaiaroto. Thank you for your patience. We released new version `v1.0.0-rc.3` which supports Go version under 1.7.4 and above 1.7.0. X-Ray Go SDK will not crush when customer's go version is under 1.7.4.
